# Publish the joint states to joint_states.
joint_state_ctrlr:
  type: joint_state_controller/JointStateController
  publish_rate: 30

left_steering_ctrlr:
  joint: joint_acker_front_left
  type: effort_controllers/JointPositionController
  pid: {p: 4.0, i: 0.0, d: 1.0}
right_steering_ctrlr:
  joint: joint_acker_front_right
  type: effort_controllers/JointPositionController
  pid: {p: 4.0, i: 0.0, d: 1.0}

left_front_axle_ctrlr:
  joint: joint_wheel_front_left
  type: effort_controllers/JointVelocityController
  pid: {p: 1.5, i: 1.0, d: 0.0, i_clamp: 10.0}
right_front_axle_ctrlr:
  joint: joint_wheel_front_right
  type: effort_controllers/JointVelocityController
  pid: {p: 1.5, i: 1.0, d: 0.0, i_clamp: 10.0}
left_rear_axle_ctrlr:
  joint: joint_wheel_rear_left
  type: effort_controllers/JointVelocityController
  pid: {p: 1.5, i: 1.0, d: 0.0, i_clamp: 10.0}
right_rear_axle_ctrlr:
  joint: joint_wheel_rear_right
  type: effort_controllers/JointVelocityController
  pid: {p: 1.5, i: 1.0, d: 0.0, i_clamp: 10.0}
